ffw/UIRPC.js

```diff
@@ -441,7 +413,7 @@ FFW.UI = FFW.RPCObserver.create(
               SDL.AlertPopUp.deactivate("ABORTED");
             } else if (typeID === 25 && SDL.ScrollableMessage.active
                && (targetID === undefined || targetID === SDL.ScrollableMessage.cancelID)) {
-              SDL.ScrollableMessage.deactivate(true);
+              SDL.ScrollableMessage.deactivate('ABORTED');
```

Comment:
```suggestion
              SDL.ScrollableMessage.deactivate(true);
```

this parameter is a boolean

Reply:
Fixed 054eed300ffaa868debb193381f640baefd9a733